### ISSUE TITLE
Some SID files wont play

### ISSUE THREAD

**Issue by xlar54** [GideonZ/1541ultimate#81]
see attached sid file for example.  sidplay2/w plays it ok
[Cheers_A.zip](https://github.com/GideonZ/1541ultimate/files/3155956/Cheers_A.zip)

**Comment by Grrrolf** [comment]
As far as I know the built-in sid player does not support playing Compute!'s Sidplayer MUS data files. The sidplay2/w probably has an own built-in play routine for these files.

```
---------------------------------------------------------------------
 --- SID INFO
---------------------------------------------------------------------
file name....:  Cheers_A.sid
md5 checksum.:  bfcf16cd44fbea391f0364919c8f57b6 (new style)
file size....:  871 bytes
title........:
author.......:
released.....:
format.......:  PSID (v02)
data offset..:  $007C
---------------------------------------------------------------------
load address.:  $6A3B
init address.:  $0000
play address.:  $0000
# of songs...:  1 (default: 1)
---------------------------------------------------------------------
player.type..:  Compute!'s Sidplayer MUS data
player.......:  C64 compatible
system clock.:  Unknown
SID chip.....:  Unknown
---------------------------------------------------------------------
```

**Comment by xlar54** [comment]
This one appears to be converted from mus to psid, so maybe some important
data is missing that the players requires.

**Comment by WilfredC64** [comment]
In the coming firmware update MUS files are supported and also MUS files inside SID files can be played, like this one.

**Comment by xlar54** [comment]
Referencing Wilfred's excellent PR #84.

**Comment by xlar54** [comment]
fixed in 1.20
